contributed_definitions/nyaml/NXresolution.yaml

```diff
@@ -0,0 +1,42 @@
+category: base
+doc: |
+  Describes the resolution of a physical quantity.
+type: group
+NXresolution:
+  dimension:
+    doc: |
+      The physical dimension of the resolution, e.g.,
+      energy, momentum, time, etc.
+    unit: NX_ANY
+  type:
+    doc: |
+      The process by which the resolution was determined.
+    enumeration: [estimated, derived, calibrated, other]
+  note(NXnote):
+    doc: |
+      Additional details of the estimate or description of the calibration procedure
+  resolution(NXdata):
+    doc: |
+      The data describing the resolution in dependency of the experimental axes.
+    magnitude(NX_FLOAT):
```

Comment:
This is automatically included in NXdata:

> Standard deviations on data values as well as coordinates can be provided by [FIELDNAME_errors](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-fieldname-errors-field) fields where FIELDNAME is the name of a [DATA](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-data-field) field or an [AXISNAME](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) field.

Reply:
Right, I think I was confused because initially, `resolution` was just an `NXnumber`. Since it is of type `NXdata` now, the standard deviation in included 👍